codacy/values-production.yaml

```diff
@@ -1,4 +1,12 @@
 global:
+  github: {}
+    # if you set this provider configuration, you must save it on the admin/integrations ui.
+    # enabled: "false"
+    # app:
+    #   appId: "1234"
+    #   TODO: Add link to documentation
```

Comment:
Remove this and create a task to track that its done when the docs are available